docs/AUDIO.md

```diff
@@ -0,0 +1,146 @@
+- [Audio sensor specs - AudioSensorSpec()](#audio-sensor-specs)
+- [APIs](#apis)
+- [Steps to run audio simulation in python](#steps-to-run-audio-simulation-in-python)
+
+
+
+### - Acoustics configuration
+
+The RLRAudioPropagationConfiguration() exposes various audio configuration that can be used to modify the audio simulation. This section lists all the available configs, their data type, default values and explains what the config does
+
+
+|Config name|Data Type|Default Value|Usage|
+|-----------|---------|-------------|-----|
+| sampleRate | int | 44100 | Sample rate for the simulated audio |
+| frequencyBands | int |  4 | Number of frequency bands in the simulated audio simulation |
+| directSHOrder | int | 3 | todo sangarg : Talk to Carl |
+| indirectSHOrder | int | 1 |  todo sangarg : Talk to Carl  |
+| threadCount | int | 1 | Number of thread the audio simulation will use |
+| updateDt | float | 0.02f | Simulation time step the audio simulation will use |
+| irTime | float | 4.f | Total render time for the audio simulation |
+| unitScale | float | 1.f | Unit scale for the scene, mesh and positions are multiplied by this factor |
+| globalVolume | float | 4.f | Total initial pressure value |
+| indirectRayCount | int | 5000 | Number of indirect rays that the ray tracer will use |
+| indirectRayDepth | int | 200 | Depth of each indirect ray that the ray tracer will use |
+| sourceRayCount | int | 200 |  Number of direct rays that the ray tracer will use |
+| sourceRayDepth | int | 10 |  Depth of each direct ray that the ray tracer will use |
+| maxDiffractionOrder | int | 10 | todo sagnarg : Talk to Carl |
+| direct | bool | true | Enable contribution from the direct rays |
+| indirect | bool | true | Enable contribution from the indirect rays |
+| diffraction | bool | true | Enable diffraction for the simulation |
+| transmission | bool | false | Enable transmission of rays |
+| meshSimplification | bool | false | Simplify mesh before running the audio simulation. todo sangarg : Add which simplification algo is used|
+| temporalCoherence | bool | false | todo sangarg : Talk to Carl |
+| dumpWaveFiles | bool | false | Write the wave files for different bands. Will be writted to the AudioSensorSpec's outputDirectory |
+| enableMaterials | bool | true | Enable audio materials |
+| writeIrToFile | bool | false | Write the final impulse response to a file |
+
+
+
+### - Channel layout
+
+This section talks about the channel layout struct. The channel layout defines what the output will look like.
+
+|Config name|Data Type|Default Value|Usage|
+|-----------|---------|-------------|-----|
+| channelType | enum | [RLRAudioPropagationChannelLayoutType](#RLRAudioPropagationChannelLayoutType).Binaural | Channel type for the simulated audio |
+| channelCount | int |  2 | Number of output channels in simulated audio |
+
+
+
+- ##### RLRAudioPropagationChannelLayoutType
+
+Lets look at the available values for RLRAudioPropagationChannelLayoutType
```

Comment:
Maybe just jump straight to the struct or add a high level blurb about this layout struct's purpose to add understanding?
```suggestion
```